qvisor/src/runc/runtime/vm.rs

```diff
@@ -370,8 +370,8 @@ impl VirtualMachine {
         info!("before loadKernel");
 
         let entry = elf.LoadKernel(Self::KERNEL_IMAGE)?;
-        //elf.LoadVDSO(&"/usr/local/bin/vdso.so".to_string())?;
-        //VMS.lock().vdsoAddr = elf.vdsoStart;
+        elf.LoadVDSO(&"/usr/local/bin/vdso.so".to_string())?;
```

Comment:
this seems to be an arbitrary choice for the path to vdso.so.
Could we have it as a global constant or configurable option.
I guess, if we have inherited this from x86, it would be sufficient to have
it defined as a global constant. This way, we can refactor the code as we
introduce new features.

Reply:
done